interfaces/portal/src/app/pages/projects-overview/components/create-project-dialog/create-project-dialog.component.html

```diff
@@ -1,37 +1,60 @@
-<app-form-dialog
-  #formDialog
-  [formGroup]="formGroup"
-  [mutation]="createProjectMutation"
-  [mutationData]="formGroup.getRawValue()"
-  header="Create project"
-  i18n-header="@@create-project"
-  headerIcon="pi pi-plus"
-  proceedLabel="Create project"
-  i18n-proceedLabel="@@create-project"
+<app-fullscreen-stepper-dialog
+  #createProjectDialog
+  [dialogVisible]="currentStep() > 0"
+  (dialogVisibleChange)="currentStep.set($event ? 1 : 0)"
+  [currentStep]="currentStep()"
+  [proceedLabel]="proceedLabel()"
+  [cannotProceed]="createProjectMutation.isPending()"
+  [queryParamStep]="'create-project-step'"
+  (goBack)="goBack()"
+  (proceed)="goToNextStep()"
 >
-  <app-form-field-wrapper
-    label="Kobo Asset ID"
-    i18n-label
-    [errorMessage]="formFieldErrors()('assetId')"
-  >
-    <input
-      pInputText
-      type="text"
-      formControlName="assetId"
-      autocomplete="one-time-code"
-      class="font-mono"
+  <ng-template #header>
+    <i class="pi pi-plus me-2"></i>
+    <ng-container i18n>Create new project</ng-container>
+  </ng-template>
+  <ng-template #step>
+    <p-card>
+      <h2 i18n>Project name</h2>
+      <p
+        i18n
+        class="mb-4"
+      >
+        Add a name and description to your project to help your users find it.
+      </p>
+      <app-project-form-name #formName />
+    </p-card>
+  </ng-template>
+  <ng-template #step>
+    <p-card>
+      <h2
+        i18n
+        class="mb-4"
+      >
+        Project information
+      </h2>
+      <app-project-form-information #formInformation />
+    </p-card>
+  </ng-template>
+  <ng-template #step>
+    <p-card>
+      <h2
+        i18n
+        class="mb-4"
+      >
+        Budget
+      </h2>
+      <app-project-form-budget #formBudget />
+    </p-card>
+  </ng-template>
+  <ng-template #illustration>
```

Comment:
I'm getting this warning in the console:
```
NG02952: The NgOptimizedImage directive (activated on an <img> element with the `ngSrc="assets/create-project.svg"`) has detected that the aspect ratio of the image does not match the aspect ratio indicated by the width and height attributes. 
Intrinsic image size: 524w x 439h (aspect-ratio: 1.19). 
Supplied width and height attributes: 1024w x 1024h (aspect-ratio: 1). 
To fix this, update the width and height attributes.
```

Reply:
Nice catch, fixed.